yt/frontends/openPMD/data_structures.py

```diff
@@ -193,7 +206,7 @@ def _populate_grid_objects(self):
         self.max_level = 0
 
 
-class openPMDDataset(Dataset):
+class openPMDDataset(Dataset, openPMDBasePath):
     """
     A dataset object contains all the information of the simulation and
```

Comment:
Ideally, a data set object should only contain a single data set (for the future).